esphome/components/fs3000/sensor.py

```diff
@@ -0,0 +1,53 @@
+# initially based off of TMP117 component
+
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import i2c, sensor
+from esphome.const import (
+    CONF_ID,
+    CONF_MODEL,
+    DEVICE_CLASS_WIND_SPEED,
+    STATE_CLASS_MEASUREMENT,
+)
+
+DEPENDENCIES = ["i2c"]
+CODEOWNERS = ["@kahrendt"]
+
+fs3000_ns = cg.esphome_ns.namespace("fs3000")
+
+FS3000Model = fs3000_ns.enum("MODEL")
+FS3000_MODEL_OPTIONS = {
+    "1005": FS3000Model.FIVE,
+    "1015": FS3000Model.FIFTEEN,
+}
+
+FS3000Component = fs3000_ns.class_(
+    "FS3000Component", cg.PollingComponent, i2c.I2CDevice, sensor.Sensor
+)
+
+CONFIG_SCHEMA = cv.All(
```

Comment:
No need to wrap in `All`
```suggestion
CONFIG_SCHEMA = (
```